adoc/SLES4SAP-hana-sr-guide-PerfOpt-12.adoc

```diff
@@ -1948,12 +2095,12 @@ recovery procedure.
 ----
 
 .{testRecover}
-.  Cleanup the failed resource status of the secondary HANA database (on
+ the failed resource status of the secondary HANA database (on
   node 2) as root.
```

Comment:
To fix the validation issue:

```suggestion
Clean up the failed resource status of the secondary HANA database (on node 2) as root.
```

(The validation issue is because of the two spaces at the beginning of line 2099. But I am guess the removel of the word "cleanup" [sic] was an accident.)

Reply:
DONE. And yes you are right it was by accident :)

Reply:
Will push a new commit yet.